src/routes.tsx

```diff
@@ -86,7 +86,11 @@ const RedirectToDashboard: FC = () => {
 
 export const routes: RouteObject[] = [
   {
-    errorElement: <RoutingErrorPage />,
+    errorElement: (
+      <AnalyticsConsentProvider>
+        <RoutingErrorPage />
+      </AnalyticsConsentProvider>
+    ),
```

Comment:
Wouldn't this make more sense to be one level above `<RouterProvider />`?

Reply:
Doesn't work.